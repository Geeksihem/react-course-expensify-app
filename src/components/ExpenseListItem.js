import React from 'react'; 
import {Link} from 'react-router-dom'; 
 
// export a staless funtion 
const ExpenseListItem  = ({  dispatch,  id, description,  amount, createdAt, note } )=>(
    <div> 
<Link to={`/edit/${id}`}>  <p> dddd</p>
</Link>
    <p> {amount} -- {createdAt} </p> 
    <p> {note}</p>
 
    </div>
)

export default  ExpenseListItem   ; 
 