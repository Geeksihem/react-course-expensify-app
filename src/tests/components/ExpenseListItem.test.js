import React from 'react'; 
import {shallow} from 'enzyme'; 
import expenses from '../fixture/expenses'; 
import ExpenseListItem from '../../components/ExpenseListItem'; 



test(' shoulf render expenseLisItem correctly', ()=>{
    const wrapper = shallow (<ExpenseListItem {...expenses[0]}/>); 
expect(wrapper).toMatchSnapshot(); 
})