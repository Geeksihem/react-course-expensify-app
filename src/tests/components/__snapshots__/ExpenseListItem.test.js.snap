// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` shoulf render expenseLisItem correctly 1`] = `
<div>
  <Link
    replace={false}
    to="/edit/1"
  >
      
    <p>
       dddd
    </p>
  </Link>
  <p>
     
    195
     -- 
    0
     
  </p>
  <p>
     
  </p>
</div>
`;
